### DISCUSSION TITLE
layout reset fails

### DISCUSSION THREAD

**Discussion by wvhulle** [sveltejs/kit#6966] [Q&A]
The layout reset doesn't work for me. When I run `npm run dev`, the server doesn't even start up.

 ```
error when starting dev server:
Error: Files prefixed with + are reserved (saw src/routes/projects/[projectLabel]/processes/[processLabel]/operations/[operationLabel]/setup/ionExchange/[columnId]/[adsorbentName]/+page@[operationLabel].svelte)
```

The `[operationLabel]` route has a layout file included. Why does the reset fail?

Thanks!

**Comment by teemingc** [comment]
there's probably an underlying issue with the page in question that's throwing an error. The error message is unrelated and always displays even when it's unrelated to the filename.

**Reply by wvhulle** [reply]
The page itself renders correctly without the @ layout reset directive. There doesn't seem to be an error on the page.

**Reply by teemingc** [reply]
could it be an issue with the layout then?
I wonder if resetting to a single layout is causing an error because there is a dependency from other layouts

**Reply by dummdidumm** [reply]
@s3812497 could you clarify what you mean by "always displays"? If this error message shows up at wrong times, we should try to fix that.

**Reply by teemingc** [reply]
> @s3812497 could you clarify what you mean by "always displays"? If this error message shows up at wrong times, we should try to fix that.

I think I first noticed it when the new routes API was released and I was migrating some routes. I can't remember how exactly to reproduce it now. However, it would display if an error existed in the `.svelte` file that prevented the page from rendering, although the file name is correct.

I'll try to experiment and see if I can reproduce it if I have time this / next week.
